packages/back-end/src/models/EventModel.ts

```diff
@@ -161,6 +161,90 @@ export const getEventForOrganization = async (
   return !doc ? null : (toInterface(doc) as EventInterface<NotificationEvent>);
 };
 
+/**
+ * Get all events for an organization, and allow for pagination
+ * @param organizationId
+ * @param page  Providing 0 as a limit will return all events
+ * @param perPage
+ * @param eventTypes
+ * @param from
+ * @param to
+ * @param sortOrder
+ * @returns
+ */
+export const getEventsForOrganization = async (
+  organizationId: string,
+  page: number,
+  perPage: number,
+  eventTypes?: string[],
+  from?: string,
+  to?: string,
+  sortOrder: 1 | -1 = -1
```

Comment:
I would consider making this an object to prevent issues with arguments being in the wrong order and also provide easy default values in any order.